installing_on_kubernetes/_topics/checking_readiness.md

```diff
@@ -0,0 +1,41 @@
+## Check readiness of the MIQ pods
```

Comment:
Can we remove `MIQ` to avoid productization concerns?  Alternatively use the productized value.